README.md

```diff
@@ -23,6 +23,14 @@ It was created as an answer to deeply recursive Pyanodon recipes, which the tool
 
 Among other things YAFC has Never Enough Items, which is FNEI on steroids. In addition to showing the recipes, it shows which ones you probably want to use, and how much.
 
+## Getting Started
+1. Go to the [releases page](https://github.com/have-fun-was-taken/yafc-ce/releases)
+2. Download the zip file for your given OSX
+3. Extract the zip file to your preferred location on your computer
+4. If on OSX/Linux, make sure to make `YAFC` executable via `chmod +x ./YAFC`
```

Comment:
I'm pretty sure there package formats around that can preserve file per permissions

Reply:
True.  Agreed... I'll be honest, I'm not sure how the releases are being set up for this repo :(

Generally, I just run it locally from the repo.  But when I did download the bundle for OSX, I was not able to execute `./YAFC` in the unzipped bundle without the permissions change.

Reply:
Agreed. I guess we need to fix the build script to keep the file permissions. (I checked, after building  with build.sh the permissions are correct)

Reply:
> I'm not sure how the releases are being set up for this repo :(

So far, it's manual. I ran [build.bat](https://github.com/have-fun-was-taken/yafc-ce/blob/master/build.bat) and uploaded the results. If there are better ways, please let us know.

Reply:
I added `build.sh` too (for non Windows).  It does build the releases for Windows/Linux/OSX just fine.  And the permissions for `Build/OSX/YAFC` was just fine so... interesting.

I think the setup for now works?  I'd love to see github actions for the releases, but I've yet to dabble with github actions and .NET